runtime/doc/editorconfig.txt

```diff
@@ -3,7 +3,7 @@
 
                             NVIM REFERENCE MANUAL
 
-
+==============================================================================
```

Comment:
Sorry to be a PITA, but could you please explain this change?

Reply:
It's generated from `gen_vimdoc.lua`.